docs/shared/features/manage-releases.md

```diff
@@ -57,6 +57,40 @@ The configuration lives in your `nx.json` file under the `"release"` section.
 }
 ```
 
+### Customize changelog output
+
+Changelog render options can be passed as [an object](https://github.com/nrwl/nx/blob/36f297cf7607fa63112415a0b5d76de09193746c/packages/nx/release/changelog-renderer/index.ts#L48) under `release.changelog.projectChangelogs.renderOptions` and `release.changelog.workspaceChangelog.renderOptions` in your `nx.json` file. Below are all options with their default values for the built-in changelog renderer.
```

Comment:
Please change this to not point to a specific commit. I appreciate you want it to remain valid, but the reality is this whole section needs to be kept up to date any way in order to remain valuabale

Reply:
Valid point. I also removed the line reference since lines tend to change as well.